lib/blogs_api/user.ex

```diff
@@ -16,7 +16,7 @@ defmodule BlogsApi.User do
   end
 
   @required_params ~w(display_name image)a
-  @allowed_params  @required_params ++ ~w(email password)a
+  @allowed_params @required_params ++ ~w(email password)a
```

Comment:
Todos os parâmetros podem ficar aqui.
```suggestion
  @allowed_params ~w(email password display_name image)a
```